### PR TITLE
Rename Note extension, new r5 backports (STU3)

### PR DESCRIPTION
**Pull request by APageNHS**
Renamed file / id / title / name of the DiagnosticReport Note extension

Added new proxy extensions requested in stu2 ballot comments

### CODE REVIEW

File: structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  <contact>
+    <name value="HL7 UK" />
+    <telecom>
+      <system value="email" />
+      <value value="ukcore@hl7.org.uk" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An extension to support adding additional info supporting info the diagnostic report." />
```
Comment (ryma2fhir) [inline structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml:20]: "support adding additional info supporting info" doesn't make sense. Same in purpose.

File: structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    </element>
+    <element id="Extension.extension:reference.value[x]">
+      <path value="Extension.extension.value[x]" />
+      <min value="1" />
+      <max value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Procedure" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-DiagnosticReport" />
```
Comment (ryma2fhir) [inline structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml:98]: needs targeting base

File: structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml [hunk 3]
```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-DiagnosticReportSupportingInfo" />
+  <url value="http://hl7.org/fhir/5.0/StructureDefinition/extension-DiagnosticReport.supportingInfo" />
+  <version value="1.0.0" />
+  <name value="ExtensionUKCoreDiagnosticReportSupportingInfo" />
+  <title value="Extension UK Core Diagnostic Report Supporting Info" />
+  <status value="draft" />
```
Comment (ryma2fhir) [inline structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml:8]: Still draft?

File: structuredefinitions/Extension-UKCore-ObservationTriggeredBy.xml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    <element id="Extension.extension:observation.url">
+      <path value="Extension.extension.url" />
+      <fixedUri value="observation" />
+    </element>
+    <element id="Extension.extension:observation.value[x]">
+      <path value="Extension.extension.value[x]" />
+      <min value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Observation" />
```
Comment (ryma2fhir) [inline structuredefinitions/Extension-UKCore-ObservationTriggeredBy.xml:72]: needs changing to base

**Review by ryma2fhir — CHANGES_REQUESTED** [review]

File: structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml (hunk 2 above)
Comment (APageNHS) [reply structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml]: Done

File: structuredefinitions/Extension-UKCore-ObservationTriggeredBy.xml (hunk 4 above)
Comment (APageNHS) [reply structuredefinitions/Extension-UKCore-ObservationTriggeredBy.xml]: Done

File: structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml (hunk 3 above)
Comment (APageNHS) [reply structuredefinitions/Extension-UKCore-DiagnosticReportSupportingInfo.xml]: Made active

**Review by ryma2fhir — APPROVED** [review]

**Pull request merged**